nusantara/nusa_datasets/kopi_cc_news/kopi_cc_news.py

```diff
@@ -0,0 +1,138 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+KoPI-CC_NEWS corpus
+
+[nusantara_schema_name] = ssp
+"""
+
+import json
+from typing import List
+
+import datasets
+import zstandard as zstd
```

Comment:
In your PR, could you also propose a change to requirements.txt because of the need to use zstandard?

Reply:
done

thanks